docs/tdvt.md

```diff
@@ -120,19 +120,11 @@ TDVT uses the 'Calcs' and 'Staples' tables.
 
    ![]({{ site.baseurl }}/assets/tdvt_connection_2.png)
 
-1. Open the TDS file in a text editor and embed the password in the <connection> tag of the TDS file next to the existing 'username' value.
-   Save the file.
+1. Run `tdvt action --add_ds mydb`. Chose to generate the password file and pick the logical query config. This will create a mydb.ini file under /config and will modify your two TDS files to rename the connection and link them to the tds/mydb.password password file.
```

Comment:
A few edit suggestions: Change 'Chose' to "Choose" to generate the password file and pick the logical query config. This creates a mydb.ini file under /config and modifies your two TDS TDS files to rename the connection and ....